pkg/oc/cli/cmd/idle.go

```diff
@@ -244,12 +257,12 @@ func (o *IdleOptions) calculateIdlableAnnotationsByService(f kcmdutil.Factory) (
 		helper, ok := helpers[gk]
 		if !ok {
 			var mapping *meta.RESTMapping
-			mapping, err = mapper.RESTMapping(schema.GroupKind{Group: gv.Group, Kind: ref.Kind}, "")
+			mapping, err = o.Mapper.RESTMapping(schema.GroupKind{Group: gv.Group, Kind: ref.Kind}, "")
 			if err != nil {
 				return nil, err
 			}
 			var client resource.RESTClient
-			client, err = f.ClientForMapping(mapping)
+			client, err = o.ClientForMappingFn(mapping)
```

Comment:
issue to remove this.  it will lead to sadness.

Reply:
https://github.com/openshift/origin/issues/20312